roles/captiveportal/templates/captiveportal.ini

```diff
@@ -0,0 +1,12 @@
+[uwsgi]
+   uid = www-data
+   gid = www-data
+   http-socket = :9090
```

Comment:
{{ apache_user }} {{ captiveportal_port }} and make this file a .j2

Reply:
agreed